### ISSUE TITLE
Glasses / Spectacles

### ISSUE THREAD

**Issue by Tinendo** [feathericons/feather#793]
## Icon Request

* Icon name: glasses
* Use case: products / educational
* Screenshots of similar icons: <img width="561" alt="Bildschirmfoto 2019-12-06 um 14 29 38" src="https://user-images.githubusercontent.com/10920151/70326498-fc640200-1834-11ea-963e-c141d1c8109b.png">

**Comment by ahtohbi4** [comment]
I tried to imagine how it could look.

<img width="507" alt="Снимок экрана 2020-02-04 в 11 49 33" src="https://user-images.githubusercontent.com/3686790/73730849-93234100-4748-11ea-8954-98bb7053dbeb.png">
<img width="507" alt="Снимок экрана 2020-02-04 в 11 49 02" src="https://user-images.githubusercontent.com/3686790/73730859-961e3180-4748-11ea-8282-8f33de207c8d.png">

<img width="693" alt="Снимок экрана 2020-02-04 в 11 54 36" src="https://user-images.githubusercontent.com/3686790/73730869-9ae2e580-4748-11ea-8456-420d2d85b466.png">

**Comment by locness3** [comment]
Go submit that @ahtohbi4 !

**Comment by ahtohbi4** [comment]
Which one? Here are two on choice from. 🙂

**Comment by Tinendo** [comment]
I like the round ones a little bit more. They look a bit friendlier.

**Comment by locness3** [comment]
Yeah, i do too, but both could be added, with the round one being `glasses` (and the other one `glasses-2`
